rskj-core/src/test/java/co/rsk/peg/NonStandardErpFederationsTest.java

```diff
@@ -337,48 +326,107 @@ void createLegacyErpRedeemScript_fromNonStandardErpBuilders() {
         );
         defaultThreshold = defaultKeys.size() / 2 + 1;
 
+        BtcECKey emergency0PublicKey = BtcECKey.fromPublicOnly(Hex.decode("0216c23b2ea8e4f11c3f9e22711addb1d16a93964796913830856b568cc3ea21d3"));
+        BtcECKey emergency1PublicKey = BtcECKey.fromPublicOnly(Hex.decode("0275562901dd8faae20de0a4166362a4f82188db77dbed4ca887422ea1ec185f14"));
+        BtcECKey emergency2PublicKey = BtcECKey.fromPublicOnly(Hex.decode("034db69f2112f4fb1bb6141bf6e2bd6631f0484d0bd95b16767902c9fe219d4a6f"));
+        emergencyKeys = Arrays.asList(
+            emergency0PublicKey, emergency1PublicKey, emergency2PublicKey
+        );
+        emergencyThreshold = emergencyKeys.size() / 2 + 1;
```

Comment:
What about tests using different `standardThreshold` and `emergencyThreshold ` than n+1, for ex: negative value, zero, greater than M, etc?

Reply:
Yes, good idea. I will create another pr for that